app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/dashboard.tsx

```diff
@@ -87,25 +88,30 @@ export interface CreateFormProps {
 // This type annotation is explicit to undo TypeScript narrowing to `false`,
 // which result in errors about unused code.
 // eslint-disable-next-line @typescript-eslint/no-inferrable-types
-const EXPERIMENTAL: boolean = true
+const EXPERIMENTAL: boolean = false
```

Comment:
The docs above do not tell the reader what features we are talking about. Thus, it is really hard to review this code, as I don't understand which features we disable. If you want to keep this with explicit info what it enables, you can do similar to what we do in Rust:

```
const EXPERIMENTAL = {
    nameOfFeatureOne: true,
    nameOfFeatureTwo: false,
}
```